sass/_aesop.scss

```diff
@@ -1,375 +1,392 @@
 .aesop-component {
-    display: block;
-    margin: 60px auto;
-	
-	+ p {
-		margin-top: 30px; // overrides style in ai-core.css
-	}
-	
-    &.aesop-image-component {
-        margin: 60px 0; // overrides style in ai-core.css
-    }
-    
-	&.aesop-audio-component {
-        @include site-width;
+  display: block;
+  margin: 60px auto;
+
+  + p {
+    margin-top: 30px; // overrides style in ai-core.css
+  }
+
+  &.aesop-image-component {
+    margin: 60px 0; // overrides style in ai-core.css
+  }
+
+  &.aesop-audio-component {
+    @include site-width;
+  }
+
+  &.aesop-quote-component {
+    padding: 60px 0;
+
+    &.aesop-component-align-left,
+    &.aesop-component-align-right {
+      margin: 10px 30px;
+      padding: 20px;
+      width: 400px !important;
+
+      @media screen and (max-width: $width-tablet) {
+        float: none;
+        margin: 40px 0;
+        padding: 60px 0;
+        width: 100% !important;
+      }
     }
-	
-    &.aesop-quote-component {
-		padding: 60px 0;
-        
-        &.aesop-component-align-left,
-        &.aesop-component-align-right {
-            margin: 10px 30px;
-            padding: 20px;
-            width: 400px !important;
-            
-            @media screen and (max-width: $width-tablet) {
-                float: none;
-                margin: 40px 0;
-                padding: 60px 0;
-                width: 100% !important;
-            }
+
+    blockquote {
+      color: inherit;
```

Comment:
Rule sets should be ordered as follows: `@extends`, `@includes` without `@content`, properties, `@includes` with `@content`, nested rule sets